clients/kubernetes.go

```diff
@@ -209,7 +209,7 @@ func newKubernetes() (*KubernetesClient, error) {
 	// library
 	clientConfig, err := rest.InClusterConfig()
 	if err != nil {
-		log.Warningf("Failed to create a InternalClientSet: %v.", err)
+		log.Debugf("Failed to create a InternalClientSet: %v.", err)
```

Comment:
In case anyone is concerned with this warning getting buried and the user not getting needed info to diagnose an issue, line 217 should still print an error if all attempts to create a client config fail.

Reply:
remove log altogether.